### PR TITLE
Truncate title

### PR DESCRIPTION
**Pull request by SimonLab**
see #605

### CODE REVIEW

File: test/truncate.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const test = require('tape');
+const truncate = require('../lib/helpers/search-results-to-template-data/truncate');
+
+test('Should convert to external types', (t) => {
+  t.equal(truncate('hello there', 5), 'hello...', 'truncate with maxChar 5');
+  t.equal(truncate('hello there', 50), 'hello there', 'return all value if maxChar > value.length');
+  t.end();
+});
```
Comment (Danwhy) [inline test/truncate.test.js]: Adding a test for truncating an empty string should fix the coverage issue

**Comment by nelsonic** [comment]
@SimonLab please fix merge conflicts then re-assign. thanks! 👍

**Comment by SimonLab** [comment]
@nelsonic resolved!

**Comment by nelsonic** [comment]
@SimonLab looks good. thanks for updating. merging. 👍
